### PR TITLE
Add translations for the developer tools statistic issues

### LINKED ISSUE #18363: Missing in lokalise text from Development tools/Statistics
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

Missing in lokalise text from Development tools/Statistics (red marked in screenshot below)

Snag_3fe54cc

### Describe the behavior you expected

Add to lokalise

### Steps to reproduce the issue

1.
2.
3.
...

### What version of Home Assistant Core has the issue?

2023.10.5

### What was the last working version of Home Assistant Core?

not relevant

### In which browser are you experiencing the issue with?

_No response_

### Which operating system are you using to run this browser?

_No response_

### State of relevant entities

_No response_

### Problem-relevant frontend configuration

_No response_

### Javascript errors shown in your browser console/inspector

_No response_

### Additional information

_No response_

### LINKED ISSUE #18364: Missing in lokalise text from Development tools/Statistics part2
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

Missing in lokalise text from Development tools/Statistics part2 (red marked in screenshot below)

Snag_4003d42

### Describe the behavior you expected

Add to lokalise

### Steps to reproduce the issue

1.
2.
3.
...

### What version of Home Assistant Core has the issue?

2023.10.5

### What was the last working version of Home Assistant Core?

not relevant

### In which browser are you experiencing the issue with?

_No response_

### Which operating system are you using to run this browser?

_No response_

### State of relevant entities

_No response_

### Problem-relevant frontend configuration

_No response_

### Javascript errors shown in your browser console/inspector

_No response_

### Additional information

_No response_

### LINKED ISSUE #18508: Untranslated String
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

Hi,

there are missing translation in Statistics/Fix section.

### Describe the behavior you expected

Strings are added to Lokalise and are ready for translation.

### Steps to reproduce the issue

1. Click on "Developer Tool"
2. Go to "Statistics"
3. Click "Fix" button

Strings are displayed in English, even I have selected Czech language:

image

### What version of Home Assistant Core has the issue?

2023.11.0

### What was the last working version of Home Assistant Core?

_No response_

### In which browser are you experiencing the issue with?

Safari 17.0 (18616.1.27.111.22, 18616)

### Which operating system are you using to run this browser?

macOS Ventura 13.6

### State of relevant entities

_No response_

### Problem-relevant frontend configuration

_No response_

### Javascript errors shown in your browser console/inspector

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by silamon**
## Proposed change
Include translations for the statistical issues in the developer tools. Occasionally, links are embedded within sentences. To simplify translations and avoid unnecessary complexity, the entire sentence is now treated as a link.

I have also activated the previously disabled lit rules in these files and updated the terminology to use 'delete' instead of 'remove.'

## Type of change

- [ ] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #18508 fixes #18364 fixes #18363
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/developer-tools/statistics/dialog-statistics-adjust-sum.ts [hunk 1]
```diff
@@ -111,7 +109,9 @@ export class DialogStatisticsFixUnsupportedUnitMetadata extends LitElement {
         scrimClickAction
         escapeKeyAction
         @closed=${this.closeDialog}
-        heading="Adjust a statistic"
+        heading=${this.hass.localize(
```
Comment (bramkragten) [inline src/panels/developer-tools/statistics/dialog-statistics-adjust-sum.ts:112]:
```suggestion
        .heading=${this.hass.localize(
```

File: src/panels/developer-tools/statistics/dialog-statistics-adjust-sum.ts [hunk 2]
```diff
@@ -173,15 +174,22 @@ export class DialogStatisticsFixUnsupportedUnitMetadata extends LitElement {
...
+        <b
+          >${this.hass.localize(
+            "ui.panel.developer-tools.tabs.statistics.fix_issue.adjust_sum.statistic"
+          )}</b
+        >
+        ${this._params!.statistic.statistic_id}
       </div>
       <ha-selector-datetime
-        label="Pick a time"
+        label=${this.hass.localize(
```
Comment (bramkragten) [inline src/panels/developer-tools/statistics/dialog-statistics-adjust-sum.ts:190]:
```suggestion
        .label=${this.hass.localize(
```

File: src/panels/developer-tools/statistics/dialog-statistics-adjust-sum.ts [hunk 3]
```diff
@@ -235,35 +269,37 @@ export class DialogStatisticsFixUnsupportedUnitMetadata extends LitElement {
       </div>
 
       <ha-selector-number
-        label="New Value"
+        label=${this.hass.localize(
```
Comment (bramkragten) [inline src/panels/developer-tools/statistics/dialog-statistics-adjust-sum.ts:272]:
```suggestion
        .label=${this.hass.localize(
```

File: src/panels/developer-tools/statistics/dialog-statistics-adjust-sum.ts [hunk 4]
```diff
@@ -235,35 +269,37 @@ export class DialogStatisticsFixUnsupportedUnitMetadata extends LitElement {
...
-        @value-changed=${(ev) => {
-          this._amount = ev.detail.value;
-        }}
+        @value-changed=${this._amountChanged}
       ></ha-selector-number>
 
       <mwc-button
         slot="primaryAction"
-        label="Adjust"
+        label=${this.hass.localize(
```
Comment (bramkragten) [inline src/panels/developer-tools/statistics/dialog-statistics-adjust-sum.ts:284]:
```suggestion
        .label=${this.hass.localize(
```

File: src/translations/en.json [hunk 5]
```diff
@@ -5821,10 +5821,58 @@
...
+                "info_text_1": "The unit of this entity changed to ''{current_unit}'' which can't be converted to the previously stored unit, ''{previous_unit}''.",
+                "info_text_2": "If the historic statistic values have a wrong unit, you can update the units of the old values. The values will not be updated.",
+                "info_text_3": "Otherwise you can choose to delete all historic statistic values, and start over."
+              },
+              "adjust_sum": {
+                "title": "Adjust a statistic",
+                "no_statistics_found": "No statistics found for this period.",
+                "info_text_1": "Sometimes the statistics end up being incorrect for a specific point in time. This can mess up your beautiful graphs! Select a time below to find the bad moment and adjust the data.",
+                "pick_a_time": "Pick a time",
+                "statistic": "Statistic:",
```
Comment (bramkragten) [inline src/translations/en.json:5869]: Move the column out of the translation and in the template, or (prefer) make the statistic id a variable:

`Statistic: {statistic}`

Comment (silamon) [reply src/translations/en.json:5869]: It's being placed in bold like this:

**Statistic:** statistic_id

Therefore I had to split it. 
I can go ahead and take the bold out as well?

**Review by bramkragten — APPROVED** [review]

File: src/translations/en.json (hunk 5 above)
Comment (bramkragten) [reply src/translations/en.json:5869]: I see we have this a lot in our translation files, so am gonna merge.

**Pull request merged**